index.html

```diff
@@ -5,8 +5,184 @@
     <title>HTML Academy: Седона</title>
   </head>
   <body>
+    <header class=" page-header">
+      <nav class="navigation">
+        <li>
+
+          <a class="main-header-logo">
+            <img src="img/logo.svg" width="170" height="70" alt="Логотип сайта городка Седона">
+          </a>
+        </li>
```

Comment:
```suggestion


          <a class="main-header-logo">
            <img src="img/logo.svg" width="170" height="70" alt="Логотип сайта городка Седона">
          </a>

```
Здесь, внутри `nav` тегов `li` быть не может. Этот тег используется только внутри списка.